### PR TITLE
Tag TaylorIntegration.jl v0.1.1

### PR DESCRIPTION
**Pull request by attobot**
Repository: [PerezHz/TaylorIntegration.jl](https://github.com/PerezHz/TaylorIntegration.jl)
Release: [v0.1.1](https://github.com/PerezHz/TaylorIntegration.jl/releases/tag/v0.1.1)
Travis: [Travis Build Status](https://travis-ci.org/PerezHz/TaylorIntegration.jl/builds/243085832)
Diff: [vs v0.1.0](https://github.com/PerezHz/TaylorIntegration.jl/compare/9c3e0ca9ac2daff208a4ef2901c2becc204b6834...afbc80100e0567f511ae44db012873838a3c9728)
`requires` vs v0.1.0: no changes
cc: @PerezHz

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

### CODE REVIEW

File: TaylorIntegration/versions/0.1.1/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.5
+TaylorSeries 0.5.0
```
Comment (tkelman) [inline TaylorIntegration/versions/0.1.1/requires]: does the .coefs change require a more recent minimum version?

Comment (PerezHz) [inline TaylorIntegration/versions/0.1.1/requires]: Not really, that change was introduced in TaylorSeries v0.5.0, which was released even before TaylorIntegration v0.1.0; I just hadn't updated TaylorIntegration's code... is that what you mean?

Comment (tkelman) [inline TaylorIntegration/versions/0.1.1/requires]: but it was working fine before? or were these changed parts not covered by tests?

**Comment by PerezHz** [comment]
It was working fine before and those parts were already being tested; it's more of an aesthetic change, which aims to make the code closer to standard math notation for Taylor series coefficients

**Comment by PerezHz** [comment]
Thanks, @tkelman!

**Pull request merged**